lib/open_api_spex/schema.ex

```diff
@@ -296,21 +296,21 @@ defmodule OpenApiSpex.Schema do
   """
   defdelegate cast(schema, value, schemas), to: DeprecatedCast
 
-  @doc ~S"""
```

Comment:
why?

Reply:
Credo compained about it and it was pretty easy to fix escaping.

Reply:
This is a Credo bug, I guess for now it isn't too bad to just change it.